### ISSUE TITLE
Promise.done is deprecated.

### ISSUE THREAD

**Issue by jniggemann** [griffin-stewie/asciidoctor-preview#32]
Atom now uses ES6 Promises instead of Q. Call promise.then instead of promise.done
```
Promise.done (/usr/share/atom/resources/app.asar/src/atom-environment.js:1214:5)
Object.addPreviewForEditor (/home/nigge/.atom/packages/asciidoctor-preview/lib/asciidoctor-preview.coffee:144:10)
Object.toggle (/home/nigge/.atom/packages/asciidoctor-preview/lib/asciidoctor-preview.coffee:120:21)
atom-workspace.atom.commands.add.asciidoctor-preview:toggle (/home/nigge/.atom/packages/asciidoctor-preview/lib/asciidoctor-preview.coffee:74:26)
CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:259:29)
<unknown> (/usr/share/atom/resources/app.asar/src/command-registry.js:3:59)
```

**Comment by griffin-stewie** [comment]
@jniggemann 
Thank you for telling me about it. I will take it as soon as possible.

**Comment by griffin-stewie** [comment]
I fixed this issue on v0.3.2.
